ruby_on_rails/app_initialisation.md

```diff
@@ -122,6 +122,17 @@ create the initializer for figaro in `config/initializers/figaro.rb`:
 * Enable the default Content Security Policies in `config/initializers/content_security_policy.rb`.
   The report URI will be set later in the step of Sentry configuration.
 
+* If you're using Webpacker, let's clean up after asset precompilation
+  to reduce Heroku slug size. Add this to the `Rakefile`:
+
+  ```ruby
+  Rake::Task['assets:clean'].enhance do
+    FileUtils.remove_dir('node_modules', true)
+    FileUtils.remove_dir('vendor/javascript', true)
+    FileUtils.remove_dir('tmp/cache/webpacker', true)
+  end
```

Comment:
love the idea. hate the name. calling it the same as the old `assets:clean`, that does a different job, causes me quite some confusion. what about `webpack:clean`? Also this would be confusing because the `assets:clean` task removes is responsible of removing old, compiled, assets, not in use anymore. this task does not clean the compiled assets. So...I like the idea. I'd name it differently, to avoid confusion.

Reply:
This is not the full definition of `assets:clean` here. I use `enhance` to extend the existing task.

Reply:
Ah! Right! Then I am fine with it!